keyboards/bear_face/keymaps/default/keymap.c

```diff
@@ -0,0 +1,98 @@
+#include QMK_KEYBOARD_H
+
+#define _QWER 0
+#define _COLE 1
+#define _DVOR 2
+#define _FN1 3
```

Comment:
Especially for the default keymap, I'd rather this be an enum, so it's easier to manage: 

```suggestion
enum layers {
    _QWER,
    _COLE,
    _DVOR,
    _FN1,
};
```